python/test/unit/language/test_core.py

```diff
@@ -5619,3 +5619,86 @@ def kernel(
     kernel[(1, )](x_tri, y_tri, shape[0], BLOCK_SIZE=shape[0])
     # compare
     np.testing.assert_allclose(y_ref, to_numpy(y_tri), rtol=0.01)
+
+
+# -----------------------
```

Comment:
Not sure we want to add the test here. We can add a `third_party/amd/python/test` directory.